application/server.ts

```diff
@@ -5,12 +5,13 @@ import fs from "fs";
 import path from "path";
 import https from "https";
 import cors from "cors";
+import axios from "axios";
 
 const app = express();
 const hostname = "0.0.0.0";
 const port = 4590;
 
-const dirName = "/app";
+const dirName = "";
```

Comment:
you should return this to how it was before merging, maybe even do something like this.
```suggestion
const dirName = process.env.PRODUCTION ? "/app" : "";
```
this makes it so only when you run the docker it is /app (which is what we want since /app doesn't exist until the docker runs). when running the docker there is an enviromental variable called PRODUCTION that exists so you can use it.